tdrs-backend/tdpservice/parsers/models.py

```diff
@@ -106,10 +109,10 @@ def get_status(self):
 
         if errors is None:
             return DataFileSummary.Status.PENDING
+        elif precheck_errors.count() > 0 or self.total_number_of_records_created == 0:
+            return DataFileSummary.Status.REJECTED
```

Comment:
can we add a unit tests for this case?

Reply:
is it always true that we should reject a file if number of pre-check errors are > 0? even if number of records > 0

Reply:
To Jan's comment: absolutely, I will get that added!

To Mo's comment: In this case yes, because we are only looking for "file wide" precheck errors. For example, if a file had two headers we would reject it. The row based precheck errors aren't taken into account in that specific query.